servicex_app/servicex_app/resources/internal/get_dataset_files_since.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from datetime import datetime
+from flask_restful import reqparse
+
+from servicex_app.decorators import auth_required
+from servicex_app.models import DatasetFile
+from servicex_app.resources.servicex_resource import ServiceXResource
+
+logger = logging.getLogger(__name__)
+
+
+class GetDatasetFilesSince(ServiceXResource):
+    @auth_required
+    def get(self):
+        parser = reqparse.RequestParser()
+        parser.add_argument('cutoff', type=str, location='args', required=False)
+        args = parser.parse_args()
+        files_query = DatasetFile.query
+
+        if args.get('cutoff'):
+            cutoff_str = args['cutoff']
+
+            try:
+                if cutoff_str.endswith('Z'):
+                    cutoff_str = cutoff_str[:-1] + '+00:00'
+
+                cutoff_datetime = datetime.fromisoformat(cutoff_str)
+
+                files_query = files_query.filter(DatasetFile.created_at >= cutoff_datetime)
```

Comment:
I think this needs to be `>` since the time you would pass into this call is the timestamp for the last file the client processed